docs/downstream-builds.md

```diff
@@ -0,0 +1,68 @@
+# Specific instructions for Downstream builds
+
+## How to manually build custom images for OpenShift
+
+1. Set the image urls for your test repos, i.e
+```console
+$ export IMG=quay.io/<username>/instaslice-operator:test
+$ export IMG_DMST=quay.io/<username>/instaslice-daemonset:test
+$ export BUNDLE_IMAGE=quay.io/<username>/instaslice-bundle:test
+```
+1. Build operator and daemonset using OpenShift specifig containerfiles
+```console
+$ make docker-build-ocp
+```
+1. Push images to you test repo
+```console
+$ make docker-push
+```
+1. Build bundle using OpenShift containerfile
+```console
+$ make bundle-build-ocp
+```
+1. Push bundle to test repo
+``` console
+$ make bundle-push
+```
+## Test the images on an OpenShift cluster using emulator mode
+
+### Prerequisites
+- You have a cluster available
+- You have the RH Cert Manager operator installed through OperatorHub
+- You have operator-sdk installed
+
+1. Deploy test bundle
+```console
+$ oc new-project instaslice-system
+$ operator-sdk run bundle $BUNDLE_IMAGE -n instaslice-system
+$ oc apply -f config/rbac/instaslice-operator-scc.yaml (FIXME)
+```
+1. Enable the emulator
+Set EMULATER_MODE=true in CSV
+
+1. Create instaslice CR for fake resources
+Edit test/e2e/resources/instaslice-fake-capacity.yaml to set metadata.name to the node the controller is running on
+```console
+$ oc apply -f test/e2e/resources/instaslice-fake-capacity.yaml
+```
+1. Label the node that was used in the name of the CR
+```console
+$ oc label node/<node_name> "nvidia.com/mig.capable"="true"
+```
+1. Execute e2e tests
+```console
+$ ginkgo -v --json-report=report.json --junit-report=report.xml --timeout 20m ./test/e2e
+```
+
+# Notes
+
+Most of this is automated in make test-e2e-ocp-emulated but with upstream images.
+
+Konflux CI with build the images to catch errors.  In the future, Konflux integration tests will test images for PRs as a set to verify images urls are correct.
```

Comment:
I couldn't get this. Do we get the capability in future to run e2e tests on the images that are built from this Dockerfile when a PR is raised?

Reply:
Konflux jobs run on the images that are build in the PR.  We can control what is run and when.  I was going to work on the testing pipeline next.

Reply:
```suggestion
Konflux CI will build the images to check for any redundancy
```
`In the future, Konflux integration tests will test images for PRs as a set to verify images urls are correct.`
Apologies that I couldn't understand this note. Could you re frame it?